src/api/hub-api.ts

```diff
@@ -6,11 +6,10 @@ interface BundleGroupAPIParam {
 }
 
 export default class HubAPI {
-  private readonly defaultBaseUrl = 'https://63d6be0e-bd49-4c76-8fe5-195bb7cf88a5.mock.pstmn.io'
-
+  private readonly defaultBaseUrl = 'https://63d6be0e-bd49-4c76-8fe5-195bb7cf88a5.mock.pstmn.io/ent/api'
```

Comment:
Not sure this is a good idea. baseUrl passed from command will not contain api path

Reply:
reverted